docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<ul><li>Could you clarify that? I'm not sure why the location doesn't also give us information about the state.</li><li>There is a trailing <code>_</code> at the end of the sentence.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>I think the correct phrasing is:</p><p> </p><p>Keep in mind, here, that given the way that we've defined the **autoregressive heteroskedastic Gaussian HMM**, it is the **variance** around the _heteroskedastic autoregressive emissions_ that gives us information about the state, _not_ the location. (To see this, notice how every time the system enters into state 2, the chain stops bouncing around much.)</p><p> </p>

Reply:
Ow ok, much clearer, thanks!
And I think you mean state 3, right? It is state 2 in the graph because of zero-indexing, but in the text above you talk about states 1, 2 and 3 -- and state 3 has the lowest variance.

Reply:
<p>Yes, that is correct.</p>